packages/edit-site/src/components/dataviews/filters.js

```diff
@@ -34,27 +36,35 @@ export default function Filters( { fields, view, onChangeView } ) {
 		} );
 	} );
 
-	return view.visibleFilters?.map( ( fieldName ) => {
-		const visibleFiltersForField = filtersRegistered.filter(
-			( f ) => f.field === fieldName
-		);
-
-		if ( visibleFiltersForField.length === 0 ) {
-			return null;
-		}
+	const visibleFilters = view.visibleFilters
+		?.map( ( fieldName ) => {
+			const visibleFiltersForField = filtersRegistered.filter(
+				( f ) => f.field === fieldName
+			);
 
-		return visibleFiltersForField.map( ( filter ) => {
-			if ( OPERATOR_IN === filter.operator ) {
-				return (
-					<InFilter
-						key={ fieldName }
-						filter={ visibleFiltersForField[ 0 ] }
-						view={ view }
-						onChangeView={ onChangeView }
-					/>
-				);
+			if ( visibleFiltersForField.length === 0 ) {
+				return null;
 			}
-			return null;
-		} );
-	} );
+
+			return visibleFiltersForField.map( ( filter ) => {
+				if ( OPERATOR_IN === filter.operator ) {
+					return (
+						<InFilter
+							key={ fieldName }
+							filter={ visibleFiltersForField[ 0 ] }
+							view={ view }
+							onChangeView={ onChangeView }
+						/>
+					);
+				}
+				return null;
+			} );
+		} )
+		.filter( Boolean );
+
+	if ( visibleFilters.length > 0 ) {
+		visibleFilters.push( <ResetFilters onChangeView={ onChangeView } /> );
```

Comment:
We should probably disable the button if the filters have default values. Or even not render at all? That's a question for designers though..

Reply:
Ah, good one - disabled it at [ec9547f](https://github.com/WordPress/gutenberg/pull/55955/commits/ec9547f09cfaf1dc70909ea6e47b3c2586ffee31)